tools/dependencies.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <CoreFxStableVersion>4.3.0</CoreFxStableVersion>
-    <RoslynVersion>2.6.0-beta3-62316-02</RoslynVersion>
+    <RoslynVersion>2.8.2-beta6-62916-08</RoslynVersion>
```

Comment:
Should we use beta4 (dev15.8, instead of dev15.8p1), or do you need bleeding edge features as they come out?

Buildtools for corefx is using beta4. https://github.com/dotnet/buildtools/blob/4aa764b049d9df84e57f89d0450951be99934593/src/Microsoft.DotNet.Build.Tasks/PackageFiles/Roslyn.Common.props#L4

If we want to stick with the latest, we should consider auto-updating this version by hooking into maestro (like the other dependency updates auto PRs): https://github.com/dotnet/versions/blob/master/build-info/dotnet/roslyn/dev15.8/Latest.txt#L1

Reply:
2.8.0-beta4 should be reasonable. We can update as needed.